merg:
    extrinsic_T: [ -0.9485, 0.0522, -0.2230 ]
    extrinsic_R: [ -0.1101,   0.0251,   -0.9932,
    0.9936,   0.0111,    -0.1100, 
   0.0080,   -0.9998,    -0.0271]
